ledger/ledger-common/src/main/kotlin/net/corda/v5/ledger/common/transactions/CpkIdentifier.kt

```diff
@@ -0,0 +1,11 @@
+package net.corda.v5.ledger.common.transactions
+
+import net.corda.v5.base.annotations.CordaSerializable
+
+//TODO: Mainly from [net.corda.libs.cpiupload.endpoints.v1.CpkIdentifier] Clarify their relationship
+@CordaSerializable
+data class CpkIdentifier(
```

Comment:
Interesting question if we should expose the metadata to flows.

Reply:
I have opened https://r3-cev.atlassian.net/browse/CORE-5940 to address this.